kotlinpoet-metadata-specs-tests/src/test/kotlin/com/squareup/kotlinpoet/metadata/specs/test/KotlinPoetMetadataSpecsTest.kt

```diff
@@ -1152,6 +1152,47 @@ class KotlinPoetMetadataSpecsTest(
       }
     }
   }
+
+  @IgnoreForHandlerType(
+      reason = "JvmOverloads is not runtime retained and thus not visible to reflection.",
+      handlerType = REFLECTIVE
+  )
+  @Test
+  fun overloads() {
+    val typeSpec = Overloads::class.toTypeSpecWithTestHandler()
+
+    //language=kotlin
```

Comment:
Does Spotless require this comment?

Reply:
Nope this is just a helper so the IDE does syntax highlighting on the below code. Normally you can use a `@Language("Kotlin")` annotation, but since this is a parameter we're passing in, this comment applies it on the contents of that parameter instead